site/content/docs/5.0/helpers/visually-hidden.md

```diff
@@ -6,13 +6,16 @@ group: helpers
 aliases: "/docs/5.0/helpers/screen-readers/"
 ---
 
-Visually hide an element while still allowing it to be exposed to assistive technologies (such as screen readers) with `.visually-hidden`. Use `.visually-hidden-focusable` to visually hide an element by default, but to display it when it's focused (e.g. by a keyboard-only user). Can also be used as mixins.
+Visually hide an element while still allowing it to be exposed to assistive technologies (such as screen readers) with `.visually-hidden`. Use `.visually-hidden-focusable` to visually hide an element by default, but to display it when it's focused (e.g. by a keyboard-only user). `.visually-hidden-focusable` can also be applied to a container–thank to the use of `:focus-within`, the container will be displayed when any child element of the container receives focus.
```

Comment:
```suggestion
Visually hide an element while still allowing it to be exposed to assistive technologies (such as screen readers) with `.visually-hidden`. Use `.visually-hidden-focusable` to visually hide an element by default, but to display it when it's focused (e.g. by a keyboard-only user). `.visually-hidden-focusable` can also be applied to a container – thanks to the use of `:focus-within`, the container will be displayed when any child element of the container receives focus.
```

I also noted that you used an em-dash which is correct in this context though I am not sure if you actually use these regularly in the documentation.

Reply:
it's our style not to use spaces before/after em dashes (which is grammatically correct, but just feels very antiquated). I'm personally not a fan, but ... that's what @mdo likes ;)

Reply:
As long as it is consistent I am fine with it. _thank_ → _thanks_ still holds I think (or at least sound weird however I am not a native speaker).

Reply:
ah, missed that bit from your suggestion